prompt_toolkit/key_binding/bindings/emacs.py

```diff
@@ -414,33 +395,35 @@ def unshift_move(event: E) -> None:
         a shift + movement key press event, moves the cursor
         as if shift is not pressed.
         """
-        key = event.key_sequence[0].key
+        sequence = " ".join([seq.key for seq in event.key_sequence])
```

Comment:
using tuples made all this code surprisingly painful, so i took some creative liberty. but do lmk if you'd rather the tuples, and i'll bring em back!